cw20/base/Cargo.toml

```diff
@@ -33,22 +34,22 @@ library = []
 
 [package.metadata.scripts]
 optimize = """docker run --rm -v "$(pwd)":/code \
-  -e CARGO_TERM_COLOR=always \
   --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/rust-optimizer:0.12.5
+  cosmwasm/rust-optimizer:0.12.6
 """
 
 [dependencies]
-cosmwasm-std = "1.0.0-beta"
-cosmwasm-storage = "1.0.0-beta"
-cw-storage-plus = "0.12"
-cw0 = "0.10"
-cw2 = "0.12"
-cw20 = "0.12"
-schemars = "0.8"
-serde = { version = "1.0", default-features = false, features = ["derive"] }
-thiserror = "1.0"
+cosmwasm-std = "1.0.0"
+cosmwasm-storage = "1.0.0"
+cw-storage-plus = "0.13.4"
```

Comment:
It's usually better to use only `0.13` instead of specifying the specific patch number. This way, if the crate has a new patch deployed, the project doesn't need to be updated again - it will automatically update to `0.13.5` when you run `cargo install` for the first time or by running `cargo update`. This comment applies to all other crates below and in other `Cargo.toml` files.

Reply:
Please, check this comment as it's not resolved yet